kortex_joystick_demo/launch/kortex_joystick_demo.launch

```diff
@@ -3,12 +3,11 @@
     <arg name="robot_name" default="kinova_gen3" />
     <arg name="robot_ip_address" default="192.168.1.10"/>
```

Comment:
Wonder if we shouldn't change this value to `10.15.20.4` since this is not a Kinova driver but our demo where we command the user to change IP to this value in the initial setup.